src/utils/utils.go

```diff
@@ -94,6 +95,14 @@ func SetFileInIgnition(ignitionData []byte, filePath, fileContents string, mode
 			Mode: &mode,
 		},
 	}
+	if config.GlobalConfig.ServiceIPs != "" && filePath == "/etc/hosts" {
```

Comment:
I thought we didn't need this as I'm removing this in #42

Reply:
Ah, right. That's a leftover. Will remove. Thank you.